### PR TITLE
[ChatAI] Update Assistant

### PR DESCRIPTION
**Pull request by jiyoung-an**
- refactor create assistant method.
- add update assistant method and test code.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -21,4 +21,9 @@ export const APP_STAKING_LOCKUP_DURATION_MS = 30 * 1000 // 30 seconds
 
 export const SUPPORTED_AINFT_STANDARDS = {
   721: '721',
-}
\ No newline at end of file
+}
+
+export const AINIZE_AI_SERVICE_NAME = new Map<string, string>([
```
Comment (platfowner) [inline src/constants.ts:26]: (suggestion) more descriptive name like PROVIDER_API_AINIZE_SERVICE_MAP?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/constants.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/constants.ts]: Done!

**Comment by jiyoung-an** [comment]
Thank you for review! @platfowner now merging...

**Pull request merged**